modules/rosa-hcp-sts-accessing-a-break-glass-cred-cli.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sts-accessing-a-break-glass-cred-cli_{context}"]
+= Accessing a {hcp-title} cluster by using a break glass credential
+
+Use the new `kubeconfig` from the break glass credential to gain temporary admin access to a {hcp-title} cluster.
+
+.Prerequisites
+
+* You have access to a {hcp-title} cluster with external authentication enabled. For more information, see _Creating a {hcp-title} cluster that uses external authentication_.
+* You have installed the `oc` CLI.
+
+.Procedure
+
+. Optional: If you have not already done so, create a `kubeconfigs` directory:
```

Comment:
Should this be a prereq? 
> You have created the `kubeconfigs` directory.  If you have not already done so, use the following command to create the directory.

Reply:
No, I will rephrase to get rid of the past tense.